includes/admin/customers/customers.php

```diff
@@ -800,6 +800,7 @@ function edd_customers_view( $customer = null ) {
 						<div class="add-customer-email-wrapper">
 							<input type="hidden" name="customer-id" value="<?php echo esc_attr( $customer->id ); ?>" />
 							<?php wp_nonce_field( 'edd-add-customer-email', 'add_email_nonce', false, true ); ?>
+							<label class = "hidden" for="additional-email">Email Address</label>
```

Comment:
The label value needs to be wrapped in a WordPress translation function: https://developer.wordpress.org/themes/functionality/internationalization/

For EDD the text domain is `easy-digital-downloads`. 

This ensures people using WordPress in another language see/hear the correctly translated value.

These translation functions can (and should be) combined with WordPress sanitization functions: https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/#escaping-with-localization